src/mock/travel.js

```diff
@@ -0,0 +1,17 @@
+import {generateStartMoment, getRandomInteger} from '../utils';
+import {generateTravelPoint} from './point';
+
+const oneTravelPoint = () => {
+  return Array(getRandomInteger(getRandomInteger(2, 5)))
+    .fill(undefined)
+    .map(generateTravelPoint);
+};
+
+export const travelPoint = () => {
```

Comment:
The name is somewhat confusing. `createTravelPoint` or `generateTravelPoint` would sound better. A general rule of thumb is to use verbs for function names. The name should answer the question "What does the function do?"